src/main/webapp/style.css

```diff
@@ -1,7 +1,41 @@
+body, html {
+  font-family: Roboto, Arial, sans-serif;
+}
+
+.card {
+  box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 3px 1px -2px rgba(0, 0, 0, 0.12), 0 1px 5px 0 rgba(0, 0, 0, 0.2);
+  position: relative;
+  transition: box-shadow .25s;
+  border-radius: 2px;
+}
+
+.card .card-title {
+  font-size: 18px;
```

Comment:
style nit: make the card title font bigger. maybe make the description paragraph font smaller. I don't think the description should overshadow the code snippet section itself